paper-ieee-trans.tex

```diff
@@ -401,20 +403,19 @@ \subsection{Software Quality Procedures}
 \label{subsec:sqa}
 
 The software quality policies were initially defined in the first deliverable document of the
-project \cite{indigo-d31} and periodically reviewed to be endowed with applicable insights of 
-software engineering practices, responses to evolving needs emanated from the user communities and 
-feedback obtained from development teams. These policies cover the 1) identification
+project \cite{indigo-d31}. The policies where reviewed periodically taking into account the requirements
```

Comment:
you missed here the software engineering insights

Reply:
yes